emulate.c

```diff
@@ -240,13 +244,13 @@ static inline bool op_op_imm(struct riscv_t *rv, uint32_t insn)
     /* dispatch operation type */
     switch (funct3) {
     case 0: /* ADDI: Add Immediate */
-        rv->X[rd] = (int32_t) (rv->X[rs1]) + imm;
+        rv->X[rd] = (int32_t)(rv->X[rs1]) + imm;
```

Comment:
I don't change this intentionally, but it looks like this violates the clang-format so they are changed.  Should I recover this first?

Reply:
Do use clang-format version 12+.